### PR TITLE
[AS-820] The AdMob custom event sample app will crash when playing card ad.

### PR DESCRIPTION
**Pull request by veretku**

### CODE REVIEW

File: app/src/main/java/com/intowow/admobdemo/activity/MediationRewardedVideoCommonActivity.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ public void onClick(View arg0) {
 
     private void loadAd() {
         if (!mRewardedVideoAd.isLoaded()) {
-            mRewardedVideoAd.loadAd(AD_UNIT_ID, new AdRequest.Builder().build());
+            mRewardedVideoAd.loadAd(AD_UNIT_ID, new AdRequest.Builder().addTestDevice("2EE641C6288FF680258C13641245A21D").build());
```
Comment (hermanliang) [inline app/src/main/java/com/intowow/admobdemo/activity/MediationRewardedVideoCommonActivity.java]: Reverse change?

**Review by i2wdev — APPROVED** [review]
OK

**Review by hermanliang — APPROVED** [review]
OK

**Review by p158276 — APPROVED** [review]
OK

**Review by wayneshiau2002 — APPROVED** [review]
OK

**Pull request merged**